tests/model_registry/rbac/utils.py

```
@@ -56,40 +58,52 @@ def assert_positive_mr_registry(
-    """
-    token = get_openshift_token()
-    namespace_instance = Namespace(client=admin_client, name=model_registry_namespace)
-    svc = get_mr_service_by_label(client=admin_client, ns=namespace_instance, mr_instance=model_registry_instance)
-    endpoint = get_endpoint_from_mr_service(svc=svc, protocol=Protocols.REST)
-    return token, build_mr_client_args(rest_endpoint=endpoint, token=token, author=author)
+    for reason in ("ReplicaSetUpdated", "NewReplicaSetAvailable"):
+        LOGGER.info(f"{_log} {reason}")
+        _wait_sampler(_reason=reason)
+
+
+def create_role_binding(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)